Tip over Core Component, highlighted part doesn't sound grammatically correct.

<img width="1047" height="94" alt="Image" src="https://github.com/user-attachments/assets/11b60823-1d63-4b32-889b-2c5e5d03c983" />

Hey @vineethkuttan!

As I was working on this issue, I tried to use VS Code to run this project using `yarn windows` after a `yarn install`, but I kept getting an error,

```
× Building Solution: C:\Users\mrkus\Kushal Code\Contributions\react-native-gallery\node_modules\react-native-win...
 × Build failed with message C:\Program Files\Microsoft Visual Studio\2022\Community\MSBuild\Microsoft\VC\v170\Microsoft.Cpp.WindowsSDK.targets(46,5): error MSB8036: The Windows SDK version 10.0.22621.0 was not found. Install the required version of Windows SDK or change the SDK version in the project property pages or by right-clicking the solution and selecting "Retarget solution". [C:\Users\mrkus\Kushal Code\Contributions\react-native-gallery\node_modules\react-native-windows\Common\Common.vcxproj]. Check your build configuration.
Command failed. Re-run the command with --logging for more information
```

And this error kept repeating trying to install all the available SDKs provided by Visual Studio 2022 Community Ddition and 2022 Build Tools, and also tried the 2026 version of Visual Studio Community Edition and Build Tools, in both when I opened the project through Visual Studio Community Edition, would make me install certain packages. But yes the error kept on repeating it, and ran `npx react-native run-windows` as per the documentation [here](https://github.com/microsoft/react-native-gallery/wiki/Manual-Validation-Steps-for-RNW-Release) under **Validate Component Functionality**.

Is it that I am installing the wrong version of the SDK? Because the exact version of the SDK is not listed in the options under **Desktop Development with C++** and **WinUI Application Development**, which were both installed when I opened the project inside of the Community Edition IDE.

Here is the [build.log](https://github.com/user-attachments/files/24751652/build.log) file that has the --logging parameter's output.

It would be really wonderful for your input help on this, and I shall also continue to research and try to fix this!

--- a/NewArch/src/components/CoreComponentBadge.tsx
+++ b/NewArch/src/components/CoreComponentBadge.tsx
@@ -15,7 +15,7 @@ export function CoreComponentBadge() {
       }
       badgeTitle="Core Component"
       icon={57611}
-      description="This component is a part of React Native Windows core. No additional library must be added in order to have access to this component in a React Native Windows project."
+      description="This component is included in the React Native Windows core, so you do not need to add any extra libraries to use it in a React Native Windows project."
     />
   );
 }

